assets/.npmrc

```diff
@@ -0,0 +1 @@
+v12.4.0
```

Comment:
@manishsongirkar Now, Node version 14.x is the LTS version so can't we use it instead of old version?

Reply:
@chandrapatel have tested with all latest version of node (v15.2.1). So removed this file as our theme compile works on all the latest versions.